index.html

```diff
@@ -51,48 +55,64 @@
                   JavaScript can be extended for a variety of purposes by
                   supplementing it with additional objects; for example:
                 </p>
+
                 <li>
+
                   - Client-side JavaScript extends the core language by
                   supplying objects to control a browser and its Document Object
                   Model (DOM). For example, client-side extensions allow an
                   application to place elements on an HTML form and respond to
                   user events such as mouse clicks, form input, and page
                   navigation.
+
                 </li>
+
                 <li>
+
                   - Server-side JavaScript extends the core language by
                   supplying objects relevant to running JavaScript on a server.
                   For example, server-side extensions allow an application to
                   communicate with a database, provide continuity of information
                   from one invocation to another of the application, or perform
                   file manipulations on a server.
```

Comment:
- [ ]  Kindly consider the `ul` tag for the list items `li` you have included, so the `ul `could be parenting these items.